### DISCUSSION TITLE
I am struggling at verifying my status

### DISCUSSION THREAD

**Discussion by neondev27** [community/community#55856] [GitHub Education]
### Hi everyone,

I am a high school student at Turkey, but even if i tried 8 times, i cannot verify myself in discount request. How can i do it?

**Comment by iamreiyn** [comment]
Hello @neodev27 

If you could give us more information such as what error you are getting, we would be able to understand your situation and help you better.

**Reply by neondev27** [reply]
I was about to apply first time, i did get error about blur, i fixed it and i get error about profile, I fixed it again and i tried, and it gives me error about document isnt english, and i put it next to original, even it is true, Github no longer accepts my request, i am just taking it:
![Screenshot_20230519-091852_Gmail](https://github.com/community/community/assets/88967833/208653ba-c3fa-4ab6-93aa-8b6612276728)

So, if anything i can do about it, tell me

**Comment by samus-aran** [comment]
Hi @neondev27 👋 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

I would also suggest that you ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub **username**, **email** and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Hope this helps 👍
